service/src/main/java/com/commercetools/pspadapter/payone/IntegrationService.java

```diff
@@ -154,18 +168,35 @@ public int port() {
         return Integer.parseInt(systemProperty);
     }
 
-    private ImmutableMap<String, Object> createHealthResponse(@Nonnull final ServiceConfig serviceConfig) {
+    ImmutableMap<String, Object> createHealthResponse(@Nonnull final String version, @Nonnull final String title,
```

Comment:
not sure if it's a good idea to change the scope just for testing..

Reply:
Why do we pass now the version and title separately as params now, instead of ServiceConfig?

Reply:
I think changing the scope to package private is ok, because this simply testing a lot

Reply:
75c21d0f